AdyenComponents/Apple Pay/ApplePayComponentDelegate.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import PassKit
+
+/// Delegates methods that respond to shopper interactions with payment authorization view controller.
+public protocol ApplePayComponentDelegate: AnyObject {
+
+    /// Tells the delegate that the shopper selected a shipping address, and asks for an updated payment request.
+    func didUpdate(contact: PKContact,
```

Comment:
If you renamed as handler then `completion` is redundant now
only one of them should be enough right?

Reply:
This is how Apple's API look like, I just copied 😅